client/components/timezone/index.jsx

```diff
@@ -68,19 +68,21 @@ class Timezone extends Component {
 				<optgroup label="UTC">
 					<option value="UTC">UTC</option>
 				</optgroup>
-				{ this.renderManualUtcOffsets() }
+				{ this.props.includeManualOffsets && this.renderManualUtcOffsets() }
 			</select>
 		);
 	}
 }
 
 Timezone.defaultProps = {
 	onSelect: noop,
+	includeManualOffsets: true,
 };
 
 Timezone.propTypes = {
 	selectedZone: PropTypes.string,
 	onSelect: PropTypes.func,
+	includeManualOffsets: PropTypes.bool,
```

Comment:
It's probably worth adding this to the README: https://github.com/Automattic/wp-calypso/tree/master/client/components/timezone

Reply:
Good point, added.